jupyterhub/tests/browser/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from playwright.async_api import async_playwright
+
+
+@pytest.fixture()
+async def browser():
+    # browser_type in ["chromium", "firefox", "webkit"]
```

Comment:
Is this just a reminder of which browsers are available? Or are you intending to support all three?

Reply:
In the future, we could make this controllable by an environment variable or CLI arg